contrib/pyln-client/pyln/client/lightning.py

```diff
@@ -1156,6 +1156,20 @@ def openchannel_abort(self, channel_id):
         }
         return self.call("openchannel_abort", payload)
 
+    def stfu_channels(self, channel_ids):
+        """ STFU multiple channels """
+        payload = {
+            "channel_ids": channel_ids,
+        }
+        return self.call("stfu_channels", payload)
+
+    def abort_channels(self, channel_ids):
+        """ Abort multiple channels """
+        payload = {
+            "channel_ids": channel_ids,
+        }
+        return self.call("abort_channels", payload)
+
```

Comment:
You don't need these wrappers, esp. for dev commands.  It automatically handles unknown RPCs (we mainly add them so you get nicer documentation and error detection).